src/test/util/asyncSafety.ts

```diff
@@ -12,20 +12,19 @@ import { AsyncFunc, Context, Done } from "mocha";
  * @param fn The test function to run
  * @returns A safely wrapped test function
  */
-export default function asyncSafety(fn: AsyncFunc) {
+export default function asyncSafety(fn: () => Promise<void>) {
```

Comment:
Changed this to remove the @ts-ignore

Reply:
Is there any consequence to only supporting functions that don't return anything?

Reply:
We're only using this in one place AFAICT, namely the recorded test runner. If we start using it elsewhere where the promise does resolve to something, that's probably a bug since the value would be unused.